org.kde.AwesomeBump.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id": "org.kde.AwesomeBump",
+    "branch": "Release",
+    "build-options": {
+        "no-debuginfo": true,
+        "strip": true
+    },
+    "command": "awesomebump",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=system-bus",
```

Comment:
what do you need this for? you shouldn't be requesting a permission this broad, run the flatpak with --log-system-bus and then whitelist the address' you actually access.